torch/_inductor/codegen/wrapper.py

```diff
@@ -1150,6 +1151,14 @@ def define_user_defined_triton_kernel(self, kernel, configs, kwargs):
         symbols_included = {original_name}
 
         def traverse(cur_kernel):
+            unqualified_loads = {
```

Comment:
if two kernels are in the same file, you'll parse it twice here

Reply:
That's true: for every kernel in the file we'll do this exactly once. But we won't emit two imports of the same symbol used in two different kernels (if that's what you mean), as the symbol will be added to the `symbols_included` set on first import emission?